### ISSUE TITLE
Issues on Windows 10

### ISSUE THREAD

**Issue by kotval** [gethvi/iDRAC6VirtualConsoleLauncher#3]
Just encase others run into this problem. I could only get it to work on windows 10 powershell by running python3 iDrac6VirtualConsuleLauncher.py. If you just use ./iDrac6VirtualConsuleLauncher.py it will not work in powershell.

**Comment by gethvi** [comment]
Yes, this is expected, I am running it the same way. You can also create a desktop shortcut with Target: "python [path-to-script] [options..]" and start the virtual console on a specific server with two clicks.
